src/main/java/com/narrowtux/fmm/util/WindowsUtil.java

```diff
@@ -0,0 +1,20 @@
+package com.narrowtux.fmm.util;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.sun.jna.platform.win32.Advapi32Util;
+import com.sun.jna.platform.win32.Win32Exception;
+import com.sun.jna.platform.win32.WinReg;
+
+public class WindowsUtil {
+    public static Path TryGetSteamPathFromRegistry() {
```

Comment:
Can you rename this to `getSteamPathFromRegistry()`? Would follow my imaginary code style guidelines.